include/message_type.hrl

```diff
@@ -0,0 +1,17 @@
+%%------------------------------------------------------------------------
+%%
+%% This file contains message types that we shouldn't include in the 
+%% offline message flow.
+%%
+%% Also contains various constant that we need to access these items.
```

Comment:
not sure what this sentence means?

Reply:
Whoops, meant to put that hard-coded `3` value below in here.